CONTRIBUTING.md

```diff
@@ -1,171 +0,0 @@
-# Contributing
```

Comment:
Should have some content, e.g. like https://github.com/equinor/dm-docs/blob/main/CONTRIBUTING.md

Reply:
added the content above